client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -129,12 +136,71 @@ const Settings = ( {
 		return renderSecurityAccordion();
 	};
 
+	const renderContactInformationSecion = () => {
+		if (
+			domain.type !== domainTypes.REGISTERED ||
+			( ! isDomainUpdateable( domain ) && ! isDomainInGracePeriod( domain ) )
+		) {
+			return null;
+		}
+
+		const getPlaceholderAccordion = () => (
+			<Accordion
+				title="Contact information"
+				subtitle="Contact information"
+				isPlaceholder
+			></Accordion>
+		);
+
+		if ( ! domain || ! domains ) return getPlaceholderAccordion();
+
+		const getContactsPrivacyInfo = () => (
+			<ContactsPrivacyInfo
+				domains={ domains }
+				selectedSite={ selectedSite }
+				selectedDomainName={ selectedDomainName }
+			></ContactsPrivacyInfo>
+		);
+
+		const contactInformation = findRegistrantWhois( whoisData );
+
+		const { privateDomain } = domain;
+		const privacyProtectionLabel = privateDomain
+			? translate( 'Privacy protection on', { textOnly: true } )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Privacy protection: on' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-management-contact-information)